db/dbHelpers.go

```diff
@@ -16,3 +16,20 @@ func UserFromDatabase(uuid uuid.UUID) (*dbo.User, string) {
 
 	return user, constants.SUCCESS
 }
+
+func VolumeFromDatabase(uuid string) (*dbo.Volume, string) {
+	var volume *dbo.Volume = dbo.NewVolume()
+
+	result := DB.DatabaseHandle.Where("uuid = ?", uuid).First(&volume)
+	if result.Error != nil {
+		return nil, constants.DATABASE_VOLUME_NOT_FOUND
+	}
+
+	return volume, constants.SUCCESS
+}
+
+func IsVolumeEmpty(uuid uuid.UUID) (bool, error) {
+	var blockCount int64
+	err := DB.DatabaseHandle.Model(&dbo.Block{}).Where("volume_uuid = ?", uuid).Count(&blockCount).Error
+	return blockCount == 0, err
+}
```

Comment:
just a thought: if you really want to implement those methods, maybe we could implement those as methods for dbo.Volume and dbo.User? And it could be used as:

```go
volume := dbo.Volume{}
volume.Fetch(params...)
```

the same goes with user (but I haven't come up with it sooner x.x)

Reply:
Niestety implementacja funkcji w dbo.Volume powoduje import cycle.

Reply:
Nie bardzo rozumiem, jak miałby działać ten Fetch?

Reply:
Nie zależy mi specjalnie na tej konkretnej funkcji, ale w jaki inny sposób mamy sprawdzić czy Volume jest pusty (aby dokonać breaking changes) niż sprawdzić czy nie ma w nim jakichś plików / bloków?